tasks-by-users/1khansaad/02-card-hover-project/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Add all this properties in universal selector.

Reply:
done bhai